docker/graphite-api.sh

```diff
@@ -1,2 +1,7 @@
 #!/bin/sh
-exec /sbin/setuser root gunicorn -b 0.0.0.0:8000 -w 2 --timeout 120 --log-level debug graphite_api.app:app >>/var/log/graphite-api.log 2>&1
+
+[ -z "$INFLUXGRAPH_GUNICORN_LOGLEVEL" ] && export INFLUXGRAPH_GUNICORN_LOGLEVEL="debug"
```

Comment:
It is docker best practice to [declare environment variables in the compose file](https://docs.docker.com/compose/environment-variables/) so users can easily see what env variables are used and what are their defaults.

So these can be moved to `docker-compose.yml` as for example:

```
 influxgraph:
    <..>
    environment:
        GUNICORN_LOGLEVEL: 'debug'
        GUNICORN_TIMEOUT: "300"
        GUNICORN_WORKERS: "2"
````

And used as:
docker-compose up -e GUNICORN_TIMEOUT=600

Reply:
I added the declaration in the docker-compose file; that the user can get an quick overview. You can check this in the latest commit.

The variables in the shell script are only declared if they are not set as environment variable. This means that the default is set if nothing is given. 
Otherwise the container will not come up if you do it the easy way: 
docker run -d influxgraph 

Some users don't care about this advanced configuration things, especially in a small test szenario, so it should work in an easy way without the declaration of the env variables.